client/src/components/Voting/Voting.js

```diff
@@ -1,13 +1,43 @@
-import MovieCard from '../MovieCard/MovieCard'
+import { useEffect, useState } from 'react'
+import axios from 'axios'
 import styled from 'styled-components/macro'
+import MovieCard from '../MovieCard/MovieCard'
 import VotingArea from './VotingArea/VotingArea'
 
 const Voting = () => {
+  const [movies, setMovies] = useState([])
+
+  useEffect(() => {
+    const fetchMovies = async () => {
+      try {
+        const response = await axios.get(`/api/movies/voting`)
+        setMovies(response.data)
+      } catch (error) {
+        setMovies([
+          {
+            tmdbId: '550',
+          },
+          {
+            tmdbId: '600',
+          },
+        ])
```

Comment:
I'm not sure if I understand it correctly, but if there is an error do you want to display two fixed movies and give an alert and an error in the console back? I just thought that from user perspective it could be confusing that there will be displayed these two movies everytime if there is an error? I would assume that as a user you could think that everything works somehow and the alert is wrong. And do you really need an alert and an error message in the console? 🤔

Reply:
Damn it, wanted to delete the alert. Thanks!

That's true. Right now it's just some mockData so to speak.